.travis.yml

```diff
@@ -1,7 +1,10 @@
 sudo: true
-dist: trusty
+dist: xenial
```

Comment:
No official support for xenial in Travis ...

Reply:
https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system